cluster/store/db.go

```diff
@@ -214,15 +256,30 @@ func (db *localDB) Close(ctx context.Context) (err error) {
 	return db.store.Close(ctx)
 }
 
-func (db *localDB) apply(op string, updateSchema, updateStore func() error, schemaOnly bool) error {
-	if err := updateSchema(); err != nil {
-		return fmt.Errorf("%w: %s: %w", errSchema, op, err)
+func (db *localDB) apply(op string, updateSchema, updateStore func() error, schemaOnly bool, applyDbChangeFirst applyDbOrder, triggerSchemaCallback schemaCallbackTrigger) error {
```

Comment:
**thought (if-minor):**  what do you think if we collect those into a `config` struct ?

Reply:
I like this idea, it would make it neater